frontend/simple-mercari-web/src/components/MerComponent/MerComponent.tsx

```diff
@@ -10,18 +10,9 @@ interface Prop {
 
 export const MerComponent: React.FC<Prop> = (props) => {
   const [cookies] = useCookies(["token", "userID"]);
-
```

Comment:
Would it be possible to comment out this code, instead of delete it? We might need to use it again in the future